packages/aserto/src/aserto/client/directory.py

```diff
@@ -0,0 +1,32 @@
+from typing import Tuple
+
+import grpc
+
+from aserto.directory.reader.v2 import ReaderStub
+from aserto.directory.writer.v2 import WriterStub
+from aserto.directory.importer.v2 import ImporterStub
+from aserto.directory.exporter.v2 import ExporterStub
+
+class Directory:
+    def __init__(self, config) -> None:
```

Comment:
Having an untyped `config` parameter doesn't give users any information about what keys the dict is expected to have (or even that it's a dict at all).
I don't think that a dict is a good choice in cases like this.

I think the simplest and most idiomatic approach is to provide keyword args:
```python
def __init__(self, *, address: str, api_key: str, tenant_id: str, ca_cert: str) -> None:
    ...
```